examples/files/cloudwatch-install-spark.tpl

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
We should make it agnostic to the applications and use a single file that picks up all logs related to EMR.

Reply:
Will be done in the next iteration.